client/apps/edit/components/section_image_collection/index.coffee

```diff
@@ -208,6 +208,7 @@ module.exports = React.createClass
               placeholder: 'Add artwork url'
             }
               button {
+                ref: 'urlAdd'
                 className: 'esic-byurl-button avant-garde-button'
```

Comment:
Here's where you would ideally change the class by checking the component's state. For instance, instead of doing `@refs.urlAdd.classList.toggle 'is-loading'` you would do `@setState loading: true` then here you could write `className: 'esic-byurl-button avant-garde-button' + if @state.loading then ' is-loading' else ''`